src/corelibs/U2Core/src/tasks/ExternalToolRunTask.cpp

```diff
@@ -120,7 +120,12 @@ void ExternalToolRunTask::run() {
     while (!externalToolProcess->waitForFinished(1000)) {
         if (isCanceled()) {
             killProcess(externalToolProcess);
-            algoLog.details(tr("Tool %1 is cancelled").arg(toolName));
+            externalToolProcess->waitForFinished(10000);
+            if (externalToolProcess->state() == QProcess::Running) {
+                algoLog.info(tr("Unable to cancel tool %1 for 10 seconds. Stop it manually by your OS task manager.").arg(toolName));
```

Comment:
Wait a bit, you can try to kill it by yourself, just by using [QProcess::kill()](https://doc.qt.io/qt-5/qprocess.html#kill). I'm sure you should try to do it and if you failed, then you text this message to user